cai_causal_graph/time_series_causal_graph.py

```diff
@@ -306,7 +306,14 @@ def get_minimal_graph(self) -> TimeSeriesCausalGraph:
                 if (
                     minimal_cg.variables is None or variable not in minimal_cg.variables
                 ) and not minimal_cg.node_exists(variable):
-                    minimal_cg.add_node(variable)
+                    # Guaranteed there is at least one node, take the first for simplicity as they should be the same
```

Comment:
Is this true? What if both X and X lag 1 are floating?

Reply:
I know you test for -1 floating in `test_lagged_floating_nodes_correctly_added_to_minimal_graph ` but what will happen when both are there?

Reply:
Extended unit test for this but it fails. How would one suspect this to behave?

Reply:
Discussed on Slack and taking first is consistent with logic of edges too. Will add backlog ticket but updated unit test for now to be consistent.